File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/DaiConfig.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class DaiConfig{

	@SerializedName("enableServerStitchedDai")
	private boolean enableServerStitchedDai;

	public void setEnableServerStitchedDai(boolean enableServerStitchedDai){
		this.enableServerStitchedDai = enableServerStitchedDai;
	}

	public boolean isEnableServerStitchedDai(){
		return enableServerStitchedDai;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/DaiConfig.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class DaiConfig : Parcelable, Serializable {
    @SerializedName("enableServerStitchedDai")
    var isEnableServerStitchedDai = false
    override fun toString(): String {
        return "DaiConfig{" +
                "enableServerStitchedDai = '" + isEnableServerStitchedDai + '\'' +
                "}"
    }
}